stub.c

```diff
@@ -37,6 +37,20 @@
 static const char __attribute__((used)) magic[] =
 	"#### LoaderInfo: stubby " GIT_VERSION " ####";
 
+BOOLEAN use_shell_cmdline(UINTN len)
+{
+	EFI_STATUS err;
+	UINTN i;
+
+	// If cmdline file is missing then use the shell's
+	if (len == 0)
+		return TRUE;
+
+	// Otherwise require LoaderIgnoreStubCmdline UEFI var to be 1
+	err = efivar_get_int(L"LoaderIgnoreStubCmdline", &i);
```

Comment:
I think this change is fine, but the variable name seems a bit long/redundant.  I'll mark it as accepted, but if you want to tweak the variable name I think that would be a good thing.  As an example, why include both "Loader" and "Stub"?  Also, even though the variable uses the stubby GUID and the global GUID, it might be best to use "Stubby" instead of "Stub" since there are a handful of things that are "EFI stubs" these days.

How about "StubbyIgnoreCmdlineSection"?

Reply:
Thanks.  StubbyIgnoreCmdlineSection  sounds good.  I'll change it.